api/src/services/commentService.ts

```diff
@@ -65,7 +101,21 @@ export default class CommentService {
             Content: content,
         } as CommentEntity);
 
-        return CommentParser.parseComment(commentEntity);
+        const comment = CommentParser.parseComment(commentEntity);
+
+        if (parentComment) {
+            const parentUserId = parentComment.userId as number;
+
+            const notificaton = new ReplyNotification(
+                this.notificationRepository,
+                comment,
+                parentUserId
+            );
+
+            notificaton.saveNotification(parentUserId);
```

Comment:
Seems redundant to pass the user id both in saveNotification and in the constructor. Why not keep it only in the constructor?

Reply:
O userId no método é o id do usuário que receberá a notificação. As informações no construtor são mais para criar o texto da notificação.
Vou remover o parentUserId do construtor e deixar só no saveNotification

Reply:
Mas se você deixar só no saveNotification, vai ter que passar pra todos os métodos abstratos como parâmetro, né?
Me parece mais fácil deixar no construtor, já que todo tipo de notificação vai precisar do ID de quem vai receber. Mas aprovaria o PR de qualquer uma das duas formas